biotools.py

```diff
@@ -0,0 +1,62 @@
+import sys
+import os
+from typing import Dict, List, Union
+
+import modules.dna_rna_tools as dna_rna
+import modules.protein_tool as protein
+import modules.fastqc_filter as fasta
+
+
+
+def fastqc_filter(input_path: str, output_filename: None = None, 
```

Comment:
```suggestion
def fastq_filter(input_path: str, output_filename: None = None, 
```
Ниже прокомментировал это